strict-tuple.cabal

```diff
@@ -26,7 +26,9 @@ source-repository head
 
 library
   build-depends:
-      base ^>= 4.11 || ^>= 4.12
+      base >= 4.11 && < 4.13
+    , bifunctors ^>= 5.5
+    , comonad ^>= 5.0
```

Comment:
Why comonad?

Reply:
Oh dear... we're in awkward Haskell type class problem land now. @emilypi your issue #1 only mentioned `Foldable`, `Traversable`, and `Bifunctor` (all in base) which are total no-brainers. How big of a dependency is `comonad` these days? I suppose the right thing to do is accept the dependency and instances, but I admit it hurts a little bit (does anyone really want `comonad` on these...?)

Reply:
(An aside: it would be nice if there was a minimal two-line `comonad-class` package to depend on instead)

Reply:
@mitchellwrosen I'll get rid of it. The cost of these extra deps for a single instance isn't worth it.